File: BudgetApp.Client/src/app/entities/Expense.ts
export class Expense {
  expenseId: string;
  name: string;
  amount: number;
  sortOrder: number;

  constructor(expenseId: string, name: string, amount: number, sortOrder: number) {
    this.expenseId = expenseId;
    this.name = name;
    this.amount = amount;
    this.sortOrder = sortOrder;
  }
}

File: BudgetApp.Domain/Entities/Expense.cs
namespace BudgetApp.Domain.Entities
{
    public class Expense
    {
        public Guid ExpenseId { get; set; }
        public string Name { get; set; } = string.Empty;
        public decimal Amount { get; set; }
        public int SortOrder { get; set; }
    }
}